RHEL6_7/system/yaboot/solution.txt

```diff
@@ -1,4 +1,4 @@
 The module is valid only on the 64-bit PowerPC architecture and it checks if Yaboot is installed.
 
-Yaboot cannot boot the Red Hat Enterprise Linux 7 system. The Red Hat Upgrade Tool will install GRUB 2 during the upgrade. Create the /etc/default/grub file before running `redhat-upgrade-tool` to ensure the correct GRUB 2 configuration. A misconfiguration might make the system unbootable.
+Yaboot cannot boot the Red Hat Enterprise Linux 7 system. Red Hat Upgrade Tool will install GRUB 2 during the upgrade. Create the /etc/default/grub file before running `redhat-upgrade-tool` to ensure the correct GRUB 2 configuration. A misconfiguration might make the system unbootable.
```

Comment:
1) I'm simply curious. Why not having _the_ here? We commonly use "The Preupgrade Assistant" for example.
2) Please replace the backticks with single quote marks.